client/src/app/videos/+video-watch/comment/video-comment.model.ts

```diff
@@ -28,6 +30,8 @@ export class VideoComment implements VideoCommentServerModel {
     this.videoId = hash.videoId
     this.createdAt = new Date(hash.createdAt.toString())
     this.updatedAt = new Date(hash.updatedAt.toString())
+    this.deletedAt = null === hash.deletedAt ? null : new Date(hash.deletedAt.toString())
```

Comment:
nitpick: I'm not sure this equal checker is need.
'has.deletedAt ? new Date(hash.deletedAt.toString()) : null' should do the same.

Reply:
thanks for the review :)